metricflow/dataset/sql_dataset.py

```diff
@@ -122,30 +122,33 @@ def column_association_for_dimension(
 
         return column_associations_to_return[0]
 
-    def column_association_for_time_dimension(
-        self,
-        time_dimension_spec: TimeDimensionSpec,
-    ) -> ColumnAssociation:
-        """Given the name of the time dimension, return the set of columns associated with it in the data set."""
+    def instances_for_time_dimensions(
```

Comment:
The pluralized version of this is not currently used but will be used further up the stack.